exo-go/src/process_helpers/index.go

```diff
@@ -38,6 +40,17 @@ func Start(dir string, commandWords ...string) (*exec.Cmd, io.WriteCloser, *byte
 	return cmd, in, &out, nil
 }
 
+// Wait waits for the given text for the specified duration
+func Wait(stdout fmt.Stringer, text string, done func()) {
```

Comment:
You meant something like this?
```
stdout, err := cmd.StdoutPipe()
if err != nil {
    return cmd, &out, err
}
scanner := bufio.NewScanner(stdout)
for scanner.Scan() {
    output := scanner.Text()
    if strings.Contains(output, text) {
       done()
    }
}
```
If so, we will need to modify `processHelpers.Start` and all the tests. One issue with pipes is that there isn't way (that I know of) to get the entire output of the command like `stdoutBuffer.String()`. Once you read it, the output is gone. We could have a global buffer when we run the test and keep writing to the buffer like [here](https://github.com/Originate/exosphere/blob/f3f6f5a6991a78faa27fb362d4c2a9fd76c92975/exo-go/src/docker_compose/index.go#L20) but that's kind of ugly. Or is there a better way?